test/commons/dom/get-composed-parent.js

```diff
@@ -0,0 +1,72 @@
+/* global xit */
+describe('dom.getComposedParent', function () {
+  'use strict';
+  var getComposedParent = axe.commons.dom.getComposedParent;
+  var fixture = document.getElementById('fixture');
+  var shadowSupport = document.body && typeof document.body.attachShadow === 'function';
```

Comment:
I know it's short, but this line seems like something we could abstract into a common test utility.